man/systemd.service.xml

```diff
@@ -414,11 +414,9 @@
         <term><varname>ExecStartPre=</varname></term>
         <term><varname>ExecStartPost=</varname></term>
         <listitem><para>Additional commands that are executed before
-        or after the command in <varname>ExecStart=</varname>,
-        respectively. Syntax is the same as for
-        <varname>ExecStart=</varname>, except that multiple command
-        lines are allowed and the commands are executed one after the
-        other, serially.</para>
+        or after the command in <varname>ExecStart=</varname>, respectively.
+        This argument takes multiple command lines, following the same scheme as described for
+        <varname>ExecStart=</varname> above.</para>
```

Comment:
Hmm, it seems better to just point people to "Command Lines"? That's the actual scheme anyway?

Reply:
Added a new commit that does this where it seemed appropriate.